src/components/Chatbot/Chatbot.tsx

```diff
@@ -0,0 +1,334 @@
+/* eslint-disable no-nested-ternary */
+
+'use client';
+
+import { Button, Card, CardBody, CardFooter, CardHeader, Divider, Tooltip } from '@nextui-org/react';
+import clsx from 'clsx';
+import { Bot, Maximize2, Minimize2, PanelLeftClose, PanelLeftOpen, Send, X } from 'lucide-react';
```

Comment:
pls use [iconsax-react](https://www.npmjs.com/package/iconsax-react)🙂

Reply:
For reasons of respecting the design, some of the icons can't be found in this library so I'm using lucide-react, otherwise I'd be using iconsax-react!